internal/cli/cmd/connectivity.go

```diff
@@ -120,6 +121,9 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	cmd.Flags().BoolVarP(&params.PauseOnFail, "pause-on-fail", "p", false, "Pause execution on test failure")
 	cmd.Flags().BoolVar(&params.SkipIPCacheCheck, "skip-ip-cache-check", true, "Skip IPCache check")
+	cmd.Flags().BoolVar(&params.Perf, "perf", false, "Run network Performance tests")
+	cmd.Flags().IntVar(&params.Duration, "perf-duration", 30, "Duration for the Performance test to run, Default:30s")
```

Comment:
```suggestion
	cmd.Flags().DurationVar(&params.PerfDuration, "perf-duration", 30*time.Second, "Duration for the Performance test to run")
```

Let's change this parameter to type `time.Duration` which has the advantage that e.g. `connectivity test --perf --perf-duration 30s` works. This is also in line with other parametes taking a duration, e.g. `--post-test-sleep`).

Also there is no need to duplicate the default value in the usage message, the auto-generated help will already include it:

```
cilium connectivity test -h
[...]
      --perf-duration int          Duration for the Performance test to run, Default:30s (default 30)
[...]
```

After applying the changes above this will show:

```
      --perf-duration duration     Duration for the Performance test to run (default 30s)
```

Reply:
Nice, wasn't aware of this! Should of looked for the reference! Thanks!